known-issues.yml

```diff
@@ -399,69 +382,67 @@ zib-Patient:
     Patient.name*:
-      - datatype: HumanName
+      - datatype: HumanName instead of a reference
         reason: A name in FHIR is represented using the HumanName datatype, not as a separate resource.
     Patient.name:
       - cardinality: 0..* instead of 0..1
         reason: The name information according to zib NameInformation may be split up over multiple instances of Patient.name.
     Patient.telecom*:
-      - datatype: ContactPoint
+      - datatype: ContactPoint instead of a reference
         reason: ContactInformation in FHIR is represented using the ContactPoint datatype, not as a separate resource.
     Patient.telecom:
       - cardinality: 0..* instead of 0..1
         for: NL-CM:0.1.5 (Patient.ContactInformation)
         reason: The cardinality mismatch between the zib (0..1) and FHIR (0..*) is explained by the missing root element of zib part ContactInformation in FHIR. Instead, the two containers of the zib (TelephoneNumbers and EmailAddresses), which both have a cardinality of 0..*, are mapped directly into the resource. Thereby this mapping is still honoring the cardinality requirements of the zib.
     Patient.address:
-      - datatype: Address
+      - datatype: Address instead of a reference
         reason: An address in FHIR is represented using the Address datatype, not as a separate resource.
     Patient.deceased[x]:deceasedDateTime:
-      - short: DateOfDeath
-        reason: This element is mapped to two zib concepts, DateOfDeath and DeathIndicator. However, the latter is implicit -- although the zib recognizes both concepts, FHIR supports populating just one of them, thus DeathIndicator is assumed to be true when DateOfDeath is populated. For clarity reasons, the metadata of deceasedDateTime is only mapped to DateOfDeath.
-      - datatype: dateTime
+      - short: DateOfDeath instead of DateOfDeath / DeathIndicator
+        reason: This element is mapped to two zib concepts, DateOfDeath and DeathIndicator. However, the latter is implicit -- although the zib recognizes both concepts, FHIR supports populating just one of them, thus DeathIndicator is assumed to be true when DateOfDeath is populated. For clarity reasons, the .deceasedDateTime slice has only DateOfDeath mapped to it.
+      - datatype: dateTime instead of boolean
         reason: The boolean expected by DeathIndicator is only assumed; see the explanation above.
     Patient.contact:
-      - datatype: BackboneElement
+      - datatype: BackboneElement instead of a reference
         reason: The zib ContactPerson _may_ be represented as Patient.contact. There is a different use case as well, where the ContactPerson is implemented using a RelatedPerson resource.
     Patient.contact.extension:contactPerson.value[x]:
-      - datatype: Reference
+      - datatype: Reference instead of rootconcept of zib ContactPerson
```

Comment:
Could we change 'rootconcept' to 'root concept', or is it mentioned like this in the qa validation message? If changing this would result in new errors, just ignore this comment. ;)

Reply:
No, this is just an informal comment. It would be even better to change it to "the root concept of"